core/src/components/picker-column/picker-column.tsx

```diff
@@ -87,13 +87,16 @@ export class PickerColumn implements ComponentInterface {
    * height of 0px.
    */
   componentWillLoad() {
+    const parentEl = (this.parentEl = this.el.closest('ion-picker') as HTMLIonPickerElement | null);
```

Comment:
Suggestion: Add a comment of the difference between this `parentEl` and `this.parentEl`. If a dev has limited time, they might not be able to quickly determine the differences.

Reply:
Good idea. Added in https://github.com/ionic-team/ionic-framework/pull/29101/commits/36966c61a46014c958362043333b5a15bedb5744